router/base-http/src/main/scala/io/buoyant/router/http/AddForwardedHeader.scala

```diff
@@ -42,13 +43,35 @@ class AddForwardedHeader(byLabel: () => String, forLabel: () => String) extends
   }
 }
 
-object AddForwardedHeader {
+class H2AddForwardedHeader(byLabel: () => String, forLabel: () => String) extends SimpleFilter[H2Request, H2Response] {
```

Comment:
These filters are protocol specific so they should be moved to router/http and router/h2 respectively.